manifests/kustomize/base/kustomization.yaml

```diff
@@ -13,16 +13,16 @@ resources:
 
 images:
 - name: gcr.io/ml-pipeline/api-server
-  newTag: 0.1.40
+  newTag: 0.2.0
```

Comment:
Shall we also bump the version under manifests/kustomize/README.md? Also the one in kubeflow/website?